src/libexpr/eval.hh

```diff
@@ -582,25 +582,41 @@ struct EvalSettings : Config
     Setting<Strings> nixPath{
         this, {}, "nix-path",
         R"(
-          List of directories to be searched for `<...>` file references.
+          List of directories to be searched for file references in `<...>` search path expressions.
+          It is overridden by the [`NIX_PATH`](@docroot@/command-ref/env-common.md#env-NIX_PATH)
+          environment variable, and can be extended using the [`-I` option](@docroot@/command-ref/opt-common.md#opt-I).
 
-          If [pure evaluation](#conf-pure-eval) is disabled,
-          this is initialised using the [`NIX_PATH`](@docroot@/command-ref/env-common.md#env-NIX_PATH)
-          environment variable, or, if it is unset and [restricted evaluation](#conf-restrict-eval)
-          is disabled, a default search path including the user's and `root`'s channels.
+          When nothing is specified:
+
+          - If [pure evaluation](#conf-pure-eval) is enabled, search paths are not resolved at all.
```

Comment:
Nitpick: it's not the search path that isn't resolved, but the `<...>` expressions.

Reply:
The problem is that these `<...>` expressions are also called search paths, at least the parser says SPATH, and IIRC the commit messages around it say "search path". Here I tried to make explicit the difference between the actual search paths and the "search path expression", but in that line I was just sloppy.

I'm fine with defining a different, more distinct name for the angle bracket expressions, but that would be another PR. Any suggestions? Am I off here about what the expression is called?